tidy3d/plugins/mode/mode_solver.py

```diff
@@ -52,6 +52,10 @@
 # Maximum allowed size of the field data produced by the mode solver
 MAX_MODES_DATA_SIZE_GB = 20
 
+# Default number of jobs (or workers) settings for parallel computing in joblib.Parallel
```

Comment:
We can basically remove reference to DEFAULT_NUM_JOB since we decided not to make any changes to the local mode solver.